pyproject.toml

```diff
@@ -21,6 +21,7 @@ pyxdg = {version = "*", platform = "linux"}
 requests = "*"
 markdown = "*"
 packaging = "*"
+pymupdf = "^1.23.6"
```

Comment:
This should be necessary only when using PyMuPDF on the host, which is not the case for us, right?

Reply:
Hehe, here's to adding it back again soon :clinking_glasses:. Accepting it for now.